packages/nextjs/components/debug/debugContractsComponent.tsx

```diff
@@ -1,15 +1,15 @@
+"use client";
+
 import { useEffect } from "react";
-import type { NextPage } from "next";
 import { useLocalStorage } from "usehooks-ts";
-import { MetaHeader } from "~~/components/MetaHeader";
 import { ContractUI } from "~~/components/scaffold-eth";
 import { ContractName } from "~~/utils/scaffold-eth/contract";
 import { getContractNames } from "~~/utils/scaffold-eth/contractNames";
 
 const selectedContractStorageKey = "scaffoldEth2.selectedContract";
 const contractNames = getContractNames();
 
-const Debug: NextPage = () => {
+export default function DebugContractsComponents() {
```

Comment:
change please to `DebugContractsComponent` and start file name with uppercase letter too. Or even better to `DebugContracts`